app/helpers/gtt_map_helper.rb

```diff
@@ -68,6 +68,13 @@ def map_tag(map: nil, layers: map&.layers,
             window.createGttClient(target);
             contentObserver();
           }, { once: true });
+          var target = document.getElementById('#{uid}');
+          if (
+            document.readyState === 'complete'
+            && !target.hasChildNodes()
+          ) {
+            window.createGttClient(target);
+          }
```

Comment:
@smellman (CC: @dkastl)
Related with #257 and PR:#258, I don't know why these changes solve the following error in #268 description.
> With this change the map will not initialize correctly when the project change triggers a dynamic loading of content.

So, reviewing these changes is really helpful. :bow:

Adding `contentObserver()` line caused to generate 2 duplicated maps when clicking on the map, so I don't include the line.